pkg/util/overridemanager/commandargsoverride.go

```diff
@@ -0,0 +1,117 @@
+		if container.(map[string]interface{})["name"] == commandRunOverrider.ContainerName {
+			commandArgsPath := fmt.Sprintf("/%s/%d/%s", specContainersPath, index, target)
+			commandArgsValue := make([]string, 0)
+			if container.(map[string]interface{})[target] == nil {
+				return nil, nil
+			}
+			for _, val := range container.(map[string]interface{})[target].([]interface{}) {
+				commandArgsValue = append(commandArgsValue, fmt.Sprintf("%s", val))
+			}
+			patch, err := acquireCommandArgsOverrideOption(commandArgsPath, commandArgsValue, commandRunOverrider)
+			if err != nil {
+				return nil, err
+			}
+			klog.V(4).Infof("[buildCommandArgsPatchesWithPath] containers patch info (%+v)", patch)
+			patches = append(patches, patch)
+		}
+	}
+	return patches, nil
+}
+
+func acquireCommandArgsOverrideOption(commandArgsPath string, commandArgsValue []string, commandOverrider *policyv1alpha1.CommandArgsOverrider) (overrideOption, error) {
+	if !strings.HasPrefix(commandArgsPath, pathSplit) {
+		return overrideOption{}, fmt.Errorf("internal error: [acquireCommandOverrideOption] commandRunPath should be start with / character")
+	}
+
+	newCommandArgs, err := overrideCommandArgs(commandArgsValue, commandOverrider)
+	if err != nil {
+		return overrideOption{}, err
+	}
+
+	klog.V(4).Infof("[acquireCommandArgsOverrideOption], newCommand info (%+v)", newCommandArgs)
+	return overrideOption{
+		Op:    string(policyv1alpha1.OverriderOpReplace),
+		Path:  commandArgsPath,
+		Value: newCommandArgs,
+	}, nil
+}
+
+func overrideCommandArgs(curCommandArgs []string, commandArgsOverrider *policyv1alpha1.CommandArgsOverrider) ([]string, error) {
+	var newCommandArgs []string
+	switch commandArgsOverrider.Operator {
+	case policyv1alpha1.OverriderOpAdd:
+		newCommandArgs = append(curCommandArgs, commandArgsOverrider.Value...)
+		klog.V(4).Infof("[overrideCommandArgs], op: %s, oldCommandArgs: %+v, newCommandArgs: %+v", policyv1alpha1.OverriderOpAdd, curCommandArgs, newCommandArgs)
+	case policyv1alpha1.OverriderOpRemove:
+		newCommandArgs = commandArgsRemove(curCommandArgs, commandArgsOverrider.Value)
+		klog.V(4).Infof("[overrideCommandArgs], op: %s, oldCommandArgs: %+v, newCommandArgs: %+v", policyv1alpha1.OverriderOpRemove, curCommandArgs, newCommandArgs)
+	default:
+		newCommandArgs = curCommandArgs
+		klog.V(4).Infof("[overrideCommandArgs], op: %s , op not supported, ignored.", policyv1alpha1.OverriderOpRemove)
+	}
+	return newCommandArgs, nil
+}
+
+func commandArgsRemove(curCommandArgs []string, removeValues []string) []string {
+	newCommandArgs := make([]string, 0, len(curCommandArgs))
+	commandArgsMap := make(map[string]struct{})
+	for _, value := range curCommandArgs {
+		commandArgsMap[value] = struct{}{}
+	}
+	for _, value := range removeValues {
+		delete(commandArgsMap, value)
+	}
+	for _, value := range curCommandArgs {
+		if _, ok := commandArgsMap[value]; ok {
+			newCommandArgs = append(newCommandArgs, value)
+		}
+	}
+	return newCommandArgs
```

Comment:
```suggestion
	currentSet := sets.NewString(curCommandArgs...)
	removedSet := sets.NewString(removeValues...)
	return currentSet.Difference(removedSet).List()
```

Reply:
`currentSet.Difference(removedSet).List()` is unordered, but `command` should be ordered.